app/controllers/transactions_controller.rb

```diff
@@ -2,22 +2,23 @@ class TransactionsController < ApplicationController
   load_and_authorize_resource :user, find_by: :name
 
   def index
-    @transactions = @user.transactions.includes(:debtor, :creditor, :issuer)
-    respond_to do |format|
-      format.json {
-        render json: @transactions.map do |t|
-          {
-            id: t.id,
-            debtor: t.debtor.name,
-            creditor: t.creditor.name,
-            time: t.updated_at,
-            amount: t.amount,
-            message: t.message,
-            issuer: t.issuer.name
-          }
-        end
+    transactions = @user.transactions.includes(:debtor, :creditor, :issuer)
```

Comment:
Are you dure removing the `@` doesn't break the HTML views?

Reply:
I don't think there are any html views. The transactions list on the index uses the `pages_controller`, and the transaction page uses the datatables endpoint.